lib/mini_profiler/profiler.rb

```diff
@@ -127,6 +127,11 @@ def user(env)
       @config.user_provider.call(env)
     end
 
+    def content_security_policy_nonce(env)
+      config_csp_nonce = @config.content_security_policy_nonce
+      config_csp_nonce.is_a?(String) ? config_csp_nonce : config_csp_nonce.call(env)
```

Comment:
I would prefer to "ultra" optimize this pipeline, can we wipe out the proc here? no need to have this extra indirection here imo: 

A simple thing like this should do the trick, no? 

`@config.content_security_policy_nonce || env["action_dispatch.content_security_policy_nonce"] || env["secure_headers_content_security_policy_nonce"]`